README.md

```diff
@@ -1,15 +1,23 @@
-# AnalyzeJournal Activity #
+AnalyzeJournal Activity
+===============
+This Activity charts data from the Sugar Journal and gives you a visual representation of Disk usage, Activity usage and Turtle data usage using a vertical bar chart, a horizontal bar chart or a pie chart.
```

Comment:
Activity does not need to be capitalised.

Reply:
Done in 14f0084